### PR TITLE
add coming soon link to v6 press article

### PR DESCRIPTION
**Pull request by ghost**
/cc @TheAlphaNerd @jasnell @mikeal

### CODE REVIEW

**Comment by ghost** [comment]
i guess this depends on when the release is actually available though? if it's within the next 5 minutes, we can let this slide

File: locale/en/blog/announcements/v6-release.md [hunk 1]
```diff
@@ -77,4 +77,4 @@ IBM, Intel, Joyent, Microsoft, PayPal and Red Hat. Gold members include GoDaddy,
 and Modulus/Progress Software, and Silver members include Apigee, AppDynamics, Codefresh, 
 DigitalOcean, Fidelity, Google, Groupon, nearForm, New Relic, npm, Opbeat, RisingStack, Sauce 
 Labs, SAP, StrongLoop (an IBM company), Sphinx, YLD!, and Yahoo!. Get involved here: 
-[http://nodejs.org](http://nodejs.org).
\ No newline at end of file
+[http://nodejs.org](http://nodejs.org).
```
Comment (MylesBorins) [inline locale/en/blog/announcements/v6-release.md]: should this be https? Also what changes here?

**Comment by MylesBorins** [comment]
+1 on this... The release will be at least another hour LGTM with small unrelated nit

File: locale/en/blog/announcements/v6-release.md (hunk 1 above)
Comment (ghost) [inline locale/en/blog/announcements/v6-release.md]: good question actually. i can add https to this real quick though :P

Comment (detailyang) [inline locale/en/blog/announcements/v6-release.md]: yep, agree with @TheAlphaNerd  https is better:)

**Comment by ghost** [comment]
alright, merging this

**Pull request merged**